packages/dashboard/src/components/widgets/primitives/input/index.css

```diff
@@ -0,0 +1,9 @@
+.input-widget-container {
+  display: flex;
+  flex-direction: row;
+}
+
+.input-widget-options {
+  flex: 1;
+  margin-right: 20px;
```

Comment:
Non-blocking: Remove magic number.

Reply:
I see your point. We only use this once so I feel creating a variable for this might be overkill.

Reply:
Maybe we should use this instead? https://cloudscape.design/components/space-between/?tabId=playground

The dashboard should do it's best to adhere to the cloudscape design system - we should definitely avoid hard coded  one off spaces where we can

Reply:
Good point @diehbria. Unfortunately with space between the select will not stretch to the width of the widget.

Reply:
What do you say we use a css variable from cloudscape instead?

Reply:
there must be some cloudscape way to put space inbetween an input and a button? this is a pretty basic use case and is the type of problem cloudscape is trying to solve with their component system.

when we hard code these values, it means the dashboard will not work with features such as 'compact' mode or other themes in the future. layout should utilize cloudscape components as much as possible, and then the design tokens if it's not possible.